nuxeo-features/rest-api/nuxeo-rest-api-server/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/adapters/EmptyDocumentAdapter.java

```diff
@@ -48,6 +55,12 @@ public DocumentModel getEmptyDocumentModel(@QueryParam("type") String type, @Que
         if (StringUtils.isBlank(type)) {
             throw new IllegalParameterException("Missing type parameter");
         }
+        // as the returned document is intended to be POSTed for creation we remove secured properties
+        Optional.ofNullable(RequestContext.getActiveContext())
+                .map(RequestContext::getRequest)
+                .map(RenderingContextWebUtils::getContext)
+                .orElseThrow(() -> new NuxeoException("No RenderingContext in the request")) // shouldn't happen
+                .addParameterValues(OMIT_PHANTOM_SECURED_PROPERTY, true);
```

Comment:
I imagine that you cannot make
```
JsonCoreIODelegate extends PartialCoreIODelegate implements InjectableProvider<Context, Type>, Injectable<RenderingContext>
```
like in the other branches?

Reply:
No the injection was introduced in nuxeo 9.x (it seems).